WISE-Community/WISE-Client
# refactor(Websocket): Replace this.upgrade.$injector('$stomp')

- Test with https://github.com/WISE-Community/WISE-API/pull/137

## Changes

Replaced ng-stomp with rx-stomp.

## Test

- Make sure websockets still work for students
  - Receive classmate work in real time (like Discussion or Peer Chat messages)
  - Receive teacher annotations (the score will not update in real time if you are on the step but it will show the new score if you leave and come back to the step)
  - Receive notifications
- Make sure websockets still work for teachers in the Classroom Monitor
  - Receive student status
  - Receive student work
  - Receive student annotations
  - Can pause student screens
- Make sure websockets still work for teachers in the Authoring Tool
  - Display other authors that are authoring the same project in the Authoring Tool

Closes #617
